tests/core/utils/test_antimeridian.py

```diff
@@ -41,44 +45,10 @@ def test_antimeridian_split() -> None:
         assert actual.equals(expected), f"actual={actual}, expected={expected}"
 
 
-def test_antimeridian_split_complicated() -> None:
```

Comment:
These cases are tested in the **antimeridian** package, so I figured it wasn't worth updating them to be correct here.

Reply:
Makes sense.